cronos-play/getting-started_unreal/quick-start/get-tokens-or-transactions.md

```diff
@@ -0,0 +1,78 @@
+# Get Tokens or Transactions
+
+## GetTokensBlocking
+
+given the BlockScout REST API base url and the account address (hexadecimal), it will return the list of all owned tokens (ref: https://cronos.org/explorer/testnet3/api-docs)
+
+<figure><img src="../../../.gitbook/assets/image (19) (2).png" alt=""><figcaption></figcaption></figure>
+
+* `Blockscout Base Url`: the base url of the BlockScout API
+* `Account Address`: the account address to query
+* `Output`: the output of the query
+* `Success`: success of the query
+* `Output Message`: error message of the query
+
+## GetTokenTransfersBlocking
+
+given the BlockScout REST API base url and the account address (hexadecimal; required) and optional contract address (hexadecimal; optional -- it can be empty if the option is ByAddress), it will return all the token transfers (ERC20, ERC721... in the newer BlockScout releases, also ERC1155) (ref: https://cronos.org/explorer/testnet3/api-docs)
+
+NOTE: QueryOption::ByContract is not supported by BlockScout
+
+<figure><img src="../../../.gitbook/assets/image (26).png" alt=""><figcaption></figcaption></figure>
+
+* `Blockscout Base Url`: the base url of the BlockScout API
+* `Address`: the account address to query
+* `Contract Address`: the contract address to query
+* `Option`: the query option
+* `Output`: the output of the query
+* `Success`: success of the query
+* `Output Message`: error message of the query \*/
+
+## GetTransactionHistoryBlocking
+
```

Comment:
I think we could put more information like:
"You need to apply Cronoscan APIKEY to activate the following functions link:"" "
@lezzokafka do you know what's the correct place the devs could apply for an APIKEY?

Reply:
Sure mate, they can check the docs [here](https://docs.cronos.org/block-explorers/block-explorer-and-api-keys#creating-api-keys-on-cronoscan) for the details. Let me add it as a suggestion

Reply:
Done